AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Organizations/OrganizationEditCommandHandler.cs

```diff
@@ -37,6 +37,17 @@ public int Handle(OrganizationEditCommand message)
 
             if (organization.Location != null)
             {
+                if (!string.IsNullOrWhiteSpace(message.Organization.Location.PostalCode))
+                {
+                    PostalCodeGeo postalCode = _context.PostalCodes.SingleOrDefault(pc => pc.PostalCode.Equals(message.Organization.Location.PostalCode, System.StringComparison.InvariantCultureIgnoreCase));
```

Comment:
a query like this exists in `CheckValidpostcodeQueryHandlerAsync`, but the handler checks the validity of a postal code using more than just the PostalCode itself, it checks the City and State as well. So I'm not too sure if it's just safe to add a postal code when it's missing without also first make sure the combination is valid